Makefile

```diff
@@ -12,6 +12,12 @@ STORAGE_INIT_IMG ?= storage-initializer:latest
 CRD_OPTIONS ?= "crd:trivialVersions=true"
 KFSERVING_ENABLE_SELF_SIGNED_CA ?= false
 
+# CPU/Memory limits for controller-manager
+CPU_LIMIT ?= 100m
+MEMORY_LIMIT ?= 300Mi
+$(shell sed -i 's/cpu:.*/cpu: $(CPU_LIMIT)/' config/default/manager_resources_patch.yaml)
```

Comment:
Can we wire this directly into the template rather than hacking it with sed?

Reply:
I don't quite understand. Which template?

Reply:
I assumed some sort of variable injection was possible with kustomize, but it looks like it's not. https://github.com/kubernetes-sigs/kustomize/blob/master/docs/eschewedFeatures.md#build-time-side-effects-from-cli-args-or-env-variables

Reply:
I remember sed could behave differently between Mac and Linux. Maybe we can use the perl command instead? https://github.com/kubeflow/kfserving/blob/master/Makefile#L122

Reply:
Since this is a simple `sed` command, it should work the same for both Mac and Linux. The only thing that might not work is the `-i` flag, which replaces the text in place. I could take out that flag and replace it with a redirect to the same patch file.

Reply:
replaced sed with perl to be more consistent with the rest of the Makefile